src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/CompilationErrorPage.cshtml

```diff
@@ -34,6 +34,10 @@
                         <div class="titleerror">@fileName</div>
                     }
                 }
+                @if (!string.IsNullOrEmpty(errorDetail.ErrorMessage))
+                {
+                    <details>@errorDetail.ErrorMessage</details>
```

Comment:
Chrome v51 and Edge seem to treat it as a regular `div` and I don't see any gestures to close it (it's visible by default).